website/common/script/cron.js

```diff
@@ -1,124 +1,124 @@
-// TODO what can be moved to /website/server?
```

Comment:
I think there's been some problem with git where it shows entire files modified even though there have been only small changes